phonenumberutil.go

```diff
@@ -2028,7 +2097,7 @@ func getMetadataForRegion(regionCode string) *PhoneMetadata {
 	if !isValidRegionCode(regionCode) {
 		return nil
 	}
-	val := regionToMetadataMap[regionCode]
+	val, _ := readFromRegionToMetadataMap(regionCode)
```

Comment:
Any reason we need to explicitly ignore the existence check of the mapping vs the old format?